resources/views/users/partials/delete-user-form.blade.php

```diff
@@ -0,0 +1,120 @@
+<section>
+    <form id="send-verification" method="post" action="{{ route('verification.send') }}">
+        @csrf
+    </form>
```

Comment:
What is this doing? 🤔

Reply:
i have literally no idea, i just copied it from what was already there

Reply:
this was copied from `update-profile-information-form.blade.php` and does not appear to do anything, so they have been removed from all forms I authored. they are still present in the codebase but I'll leave removing them up to you